x-pack/test/apm_api_integration/basic/tests/index.ts

```diff
@@ -4,9 +4,13 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 import { FtrProviderContext } from '../../common/ftr_provider_context';
+import * as matchSnapshot from '../../common/match_snapshot';
 
 export default function apmApiIntegrationTests({ loadTestFile }: FtrProviderContext) {
   describe('APM specs (basic)', function () {
+    beforeEach(matchSnapshot.init);
+    afterEach(matchSnapshot.teardown);
```

Comment:
Alternatively, we can try using require('../../common/match_snapshot'), and then calling `beforeEach()` and `afterEach()` in that file. We could also add an `after()` hook there that checks if there are unused snapshots (we'd need to track them in `expectMatchSnapshot`).